tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    py{27,34,35}-django{17,18,19,110}-alchemy10-mongoengine010
+    py{27,34,36}-django{17,18,19,110}-alchemy10-mongoengine010
```

Comment:
Beware: this is removing support for the py35 builds in the tox matrix (which is much wider than what we use in travis).